jdi-light-material-ui-tests/src/main/java/io/github/com/pages/displaydata/IconsPage.java

```diff
@@ -0,0 +1,51 @@
+package io.github.com.pages.displaydata;
+
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+
+import java.util.List;
+
+public class IconsPage extends WebPage {
+    @UI("//div[contains(@class, 'MuiGrid-grid-xs-8')][1]/*[name()='svg']")
+    public static List<UIElement> filled;
+
+    @UI("//div[contains(@class, 'MuiGrid-grid-xs-8')][2]/*[name()='svg']")
+    public static List<UIElement> outlined;
+
+    @UI("//div[contains(@class, 'MuiGrid-grid-xs-8')][3]/*[name()='svg']")
+    public static List<UIElement> rounded;
+
+    @UI("//div[contains(@class, 'MuiGrid-grid-xs-8')][4]/*[name()='svg']")
+    public static List<UIElement> twoTone;
+
+    @UI("//div[contains(@class, 'MuiGrid-grid-xs-8')][5]/*[name()='svg']")
+    public static List<UIElement> sharp;
+
+    @UI("//div[contains(@class, 'MuiGrid-grid-xs-8')][6]/*[name()='svg']")
+    public static List<UIElement> edgeCase;
+
+    @UI("#simpleLastClick")
+    public static UIElement simpleLastClick;
+
+    @UI("#simpleLastHover")
+    public static UIElement simpleLastHover;

```

Comment:
please use Text type for this variable from com.epam.jdi.light.ui.html.elements.common.Text